Plugins/Public/base_plugin/FactoryModule.cpp

```diff
@@ -32,53 +25,95 @@ wstring FactoryModule::GetInfo(bool xml)
 	if (Paused)	Status = L"(Paused) ";
 	else Status = L"(Active) ";
 
+	info += recipeMap[factoryNickname].infotext;
+
+	wstring openLine;
+	wstring closeLine;
 	if (xml)
 	{
-		info += FACTORY_NAMES[type];
-		info += L"</TEXT><PARA/><TEXT>      Pending " + stows(itos(build_queue.size())) + L" items</TEXT>";
-		if (active_recipe.nickname)
+		openLine = L"</TEXT><PARA/><TEXT>      ";
+		closeLine = L"</TEXT>";
+	}
+	else
+	{
+		openLine = L" - ";
+		closeLine = L"\n";
+	}
+	info += openLine + L"Pending " + stows(itos(build_queue.size())) + L" items";
+	if (active_recipe.nickname)
+	{
+		info += openLine + L"Crafting " + Status + active_recipe.infotext + L". Waiting for:";
```

Comment:
missing closeLine at the end here?

Reply:
The idea is that openLine already contains a terminator for the last line (ever new line is preceded after a previous line that needs closing, after all. closeLine is only used at the very end of writing the string.

I have however noted that the newline prefix was not present in non-XML variant, adding.